package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "masterstake",
+  "version": "0.0.1",
+  "description": "Masterstake",
+  "main": "index.js",
+  "scripts": {
+    "server-dev": "./node_modules/nodemon/bin/nodemon.js ./api/src/index.js",
+    "test": "./node_modules/tap/bin/run.js './{,!(node_modules)/**}/*.test.js'",
+    "start": "node ./api/src",
+    "docker-build": "./scripts/build.sh",
+    "docker-start": "./scripts/start.sh"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
```

Comment:
doesn't work

Reply:
Removed all together as we are using vscode's ```"editor.formatOnSave": true``` anyways.